spec/system/provider_interface/provider_duplicates_a_conditions_on_an_offer_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
```

Comment:
is this something we want to capture in a unit test rather than a system spec?

Reply:
will add!

Reply:
great! could you remove the system spec please, as this is now covered in the other spec?

Reply:
goneskies